### ISSUE TITLE
Change audio device

### ISSUE THREAD

**Issue by Saphant** [dbkynd/controlcast#17]
Hi,

I'm mostly interested in using my launchpad as a sound board, but I would like to be able to stream that through a VOIP. Is it possible to add functionality that would allow users to change the audio device that it's going to use?

**Comment by ruohki** [comment]
I dont think this tool is maintened so i started my own project, it does allow you switch audio output sources. No source release yet as I need to shape things but if you want to grab it you can do so in the discord I did setup for that purpose: https://discord.gg/4Ys9TRR
